### ISSUE TITLE
Crash when starting minecraft

### ISSUE THREAD

**Issue by Borgos912** [benbenlaw-modpacks/Techopolis-2#95]
Hello, I installed the modpack today to play with a friend but every time I try to open it crashes with a exit code 1 (crash log attached), minecraft launcher opens normally and choosing the 1.20.2 latest release works as expected but the forge-43.3.2 option always crashes.
I tried reinstalling the modpack and reinstalling curseforge but nothing worked.
[hs_err_pid11980.log](https://github.com/bbl-team/Techopolis-2/files/13273828/hs_err_pid11980.log)

**Comment by Darkjustdark** [comment]
i have the same issue with a simular crash log

**Comment by Gryffen1971** [comment]
I just check the Curseforge Website and Techopolis-2 is on running on MC1.19.2
Where is there a Version for MC1.20.2 located at?
Please verify that you are running the Modpack on Version 1.19.2 Forge Version for Minecraft, plz.

**Comment by Darkjustdark** [comment]
Yes i was running that version of the pack. My problem was solved by updating my drivers :)
